config/environments/production.rb

```diff
@@ -79,6 +80,19 @@
 
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
+  config.action_controller.perform_caching = true
+
+  config.cache_store = :mem_cache_store,
+                       (ENV['MEMCACHIER_SERVERS'] || '').split(','),
```

Comment:
Would be curious how to test this locally / what happens if these aren't defined. Also, what setup did you / do we have to do in heroku for staging / prod?

Reply:
in development i dont think its enabled at all

Reply:
Gotchya, that's probably for the best!